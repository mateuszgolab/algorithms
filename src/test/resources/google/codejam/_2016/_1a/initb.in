5
3
1 2 3
2 3 5
3 5 6
2 3 4
1 2 3
2
22 222
2 22
2 22
2
2 6
6 8
4 8
4
2 4 5 6
6 8 9 12
1 2 3 5
1 2 4 6
4 6 8 10
3 5 8 9
2 4 6 8
4
11 14 16 22
3 6 9 11
6 7 11 14
9 11 15 19
10 11 19 22
6 7 10 11
3 6 8 10