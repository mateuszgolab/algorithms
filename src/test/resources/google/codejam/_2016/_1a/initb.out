Case #1: 3 4 6
Case #2: 22 222
Case #3: 2 4
Case #4: 5 6 10 12
Case #5: 8 10 15 16
